velox/docs/functions/spark/string.rst

```diff
@@ -8,6 +8,11 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
 
     Returns unicode code point of the first character of ``string``. Returns 0 if ``string`` is empty.
 
+.. spark:function:: bit_length(string/binary) -> integer
+
+    Returns the bit length for the specified string column. ::
+        SELECT bit_length('123'); -- 24
```

Comment:
Would you generate the docs and verify they look good. You might need to add an empty line before SELECT.